joblib/test/test_numpy_pickle.py

```diff
@@ -245,17 +245,6 @@ def test_masked_array_persistence():
     nose.tools.assert_true(isinstance(b, np.ma.masked_array))
 
 
-def test_z_file():
```

Comment:
> restoring read_z_file/write_z_file + corresponding test functions

You probably only want to keep the reading functionality for backward compatibility.